src/page/TodoApp.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import TodoForm from '../components/TodoForm';
+import TodoHeader from '../components/TodoHeader';
+import TodoItem from '../components/TodoItem';
+import { ListGroup } from 'react-bootstrap';
+
+const TodoApp = () => {
+  const todoItems = ['A', 'B', 'C'].map((o, i) => <TodoItem key={i} todo={o} />);
+
+  return (
+    <div className="d-flex flex-column justify-content-center w-50 mx-auto my-0">
+      <TodoHeader />
+      <TodoForm />
+      <ListGroup>{todoItems}</ListGroup>
```

Comment:
直接寫到 return 裡面可以讓 react 幫忙去處理 [A, B, C] 這個 arr 的每一個物件有沒有改變，會自動判斷有沒有觸發 rerender（重複渲染），如果你自己算完再放進去，那就必定會 rerender。

```suggestion
      <ListGroup>
        {['A', 'B', 'C'].map((o, i) => <TodoItem key={i} todo={o} />)}
      </ListGroup>
```

Reply:
喔好 那這個我先像你這樣寫